File: src/main/java/com/coors/guess/Game.java
package com.coors.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int secret = getRandomNumber();
        System.out.println(secret);
        int number = 0;
        while (number != secret) {
            System.out.printf("請輸入一個數字： ");
            number = scanner.nextInt();
            if (number > secret) {
                System.out.println("要再低一點");
            } else if (number < secret) {
                System.out.println("要再高一點");
            } else {
                System.out.println("恭喜答對啦!!!");
            }
        }
    }

    private static int getRandomNumber() {
        return new Random().nextInt(10)+1;
    }
}

File: src/main/kotlin/com/coors/Game.kt
package com.coors

import java.util.*

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val secretNumber = getRandomNumber()
    var userInputNumber = 0
    while (userInputNumber != secretNumber) {
        print("請輸入1~10的數字 : ")
        userInputNumber = readLine()!!.toInt()
        if (userInputNumber > secretNumber) {
            println("再低一點")
        } else if (userInputNumber < secretNumber) {
            println("再高一點")
        } else {
            println("恭喜答對!!! $userInputNumber")
        }
    }
}

fun getRandomNumber() : Int {
    return Random().nextInt(10)+1
}